ocs_ci/utility/vsphere.py

```diff
@@ -384,3 +384,86 @@ def restart_vms(self, vms, force=True):
         """
         self.stop_vms(vms, force=force)
         self.start_vms(vms)
+
+    def is_resource_pool_exist(self, pool, dc, cluster):
+        """
+        Check whether resource pool exists in cluster or not
+
+        Args:
+            pool (str): Resource pool name
+            dc (str): Datacenter name
+            cluster (str): Cluster name
+
+        Returns:
+            bool: True if resource pool exists, otherwise False
+
+        """
+        return True if self.get_pool(pool, dc, cluster) else False
+
+    def poweroff_vms(self, vms):
```

Comment:
You should actually move this to different PR as it seems unrelated to this fix.

Reply:
`poweroff_vms` is called by `destroy_vms` which is called by `destroy_pool` which is called by `post_destroy_checks ` :)

Reply:
thanks :)